.Lib9c.Tests/Action/BattleArenaTest.cs

```diff
@@ -0,0 +1,1182 @@
+namespace Lib9c.Tests.Action
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Bencodex.Types;
+    using Libplanet;
+    using Libplanet.Action;
+    using Libplanet.Assets;
+    using Libplanet.Crypto;
+    using Nekoyume;
+    using Nekoyume.Action;
+    using Nekoyume.Arena;
+    using Nekoyume.Model;
+    using Nekoyume.Model.Arena;
+    using Nekoyume.Model.EnumType;
+    using Nekoyume.Model.State;
+    using Nekoyume.TableData;
+    using Serilog;
+    using Xunit;
+    using Xunit.Abstractions;
+    using static Lib9c.SerializeKeys;
+
+    public class BattleArenaTest
```

Comment:
**OriginActionName**Test was obsoleted. Please writing in [BattleArena8Test.cs](https://github.com/planetarium/lib9c/blob/development/.Lib9c.Tests/Action/BattleArena8Test.cs).